app/views/users/index.html.erb

```diff
@@ -1,2 +1,26 @@
-<h1>Users Index Page</h1>
-<p>Here is a list of users:</p>
+<% content_for :head do %>
+  <%= stylesheet_link_tag 'userstyle', media: 'all' %>
+<% end %>
+
+<h1>Here is a list of users:</h1>
+<ul>
+  <% @users.each do |user| %>
+  <div class ="userphotoName">
+    <div class="photo"> 
+      <% if user.photo.present? %>
+        <%= image_tag user.photo, class: "user-photo" %>
+      <% else %>
+        <%= image_tag "default_avatar.png", alt: "Default Avatar", class: "user-photo" %>
+      <% end %>
+    </div>
+    <li class="user-list">
+      <span class="user-name"><%= user.name %></span>
```

Comment:
- **[OPTIONAL]** I suggest using links for the names to be able to redirect to the user#show page.